website/docs/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb.md

```diff
@@ -63,39 +63,36 @@ chmod 600 ~/.ssh/authorized_keys
-  
-Some versions of Linux used on bastion hosts use a version of `sshd` with additional idle timeout settings `ClientAliveCountMax`. This value sets the number of client alive messages that may be sent without `sshd` receiving any messages back from the client. If this threshold is reached while client alive messages are being sent, `sshd` will disconnect the client, terminating the session. The client-alive mechanism is helpful when the client or server needs to know when a connection has become inactive. The default value is 3.<br /><br />
-
-`ClientAliveInterval`:<br />
-This value sets a timeout interval in seconds after which if no data has been received from the client, `sshd` will send a message through the encrypted channel to request a response from the client. The default is 0, indicating that these messages will not be sent to the client.<br />
-
-Using default values, tunnels could be terminated prematurely by `sshd`. To solve this problem, the `/etc/ssh/sshd_config` file on the bastion host can be configured with the following values:<br /><br />
-- `ClientAliveCountMax` 10<br />
-- `ClientAliveInterval` 30<br />
-where `ClientAliveCountMax` should be set to a non-zero value and `ClientAliveInterval` should be a value less than the ELB or NLB idle timeout value.<br />
-
-With these settings, unresponsive SSH clients will be disconnected after approximately 300 seconds, helping to prevent tunnel failures.
-</details>
-
 
 ## Configuration
 
-To learn how to optimize performance with data platform-specific configurations in dbt Cloud, refer to [Redshift-specific configuration](/reference/resource-configs/redshift-configs).
+To optimize performance with data platform-specific configurations in dbt Cloud, refer to [Redshift-specific configuration](/reference/resource-configs/redshift-configs).
 
 To grant users or roles database permissions (access rights and privileges), refer to the [Redshift permissions](/reference/database-permissions/redshift-permissions) page or [Postgres permissions](/reference/database-permissions/postgres-permissions) page.
+
+## FAQs
+
+<detailsToggle alt_header="Database Error - could not connect to server: Connection timed out">
+When setting up a database connection using an SSH tunnel, you typically need these parts:
+
+- A Load Balancer (like ELB or NLB) to manage traffic.
+- A bastion host (or jump server) that runs the SSH protocol, acting as a secure entry point.
+- The database itself (such as a Redshift cluster).
+
+dbt Cloud uses an SSH tunnel to connect through the Load Balancer to the database. This tunnel is set up at the start of any dbt job run. If the tunnel drops, the job fails.
+
+Tunnel failures usually happen because:
+
+- The SSH service stops if there's no activity for too long.
```

Comment:
The SSH service is a much broader term than SSH daemon. It might be better to be specific here unless someone on our team suggested the change. 
```suggestion
- The SSH daemon times out if it's idle for too long.
```